### PR TITLE
completed requirements and bonus

### PR DESCRIPTION
**Pull request by grutt**

### CODE REVIEW

File: squawker/server.py [hunk 1]
```diff
@@ -37,11 +38,88 @@ def close_connection(exception):
...
+
+
+@app.errorhandler(InvalidUsage)
+def handle_invalid_usage(error):
+    response = jsonify(error.to_dict())
+    response.status_code = error.status_code
+    return response
+
+# set the post length as a configuration variable
+MAX_LEN = 3
```
Comment (afeld) [inline squawker/server.py]: I think you mean `140`? 😉

Comment (grutt) [reply squawker/server.py]: lol, yes. The problems with manual testing.

**Comment by grutt** [comment]
@afeld please see latest commit on #11 
A delay is required on the pagination tests because it is looking for order and sqlite3 has 1 second resolution. 
Cheers,GR

**Comment by afeld** [comment]
Good catch! Brought your commit into `master` as a50bd7707404a58b59c4e170e058124c956825cf.

**Pull request closed without merging**